test/fixtures/api-client/ethnicity.findAll.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
there's a mix of case in the filename here is this correct? `api-client/ethnicity.findAll.json`

Reply:
See above. These all relate to the same problem so going to resolve these ones and continue the discussion on the first file.